### PR TITLE
Add nlp.py and add_sentiment() function

### PR DESCRIPTION
**Pull request by bdfsaraiva**
First blood of #60 

Note: need transformers in setup.cfg

### CODE REVIEW

File: chatminer/nlp.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+def add_sentiment(df: pd.DataFrame, lang: str = "en") -> pd.DataFrame:
+    model_path = (
+        "cardiffnlp/twitter-roberta-base-sentiment-latest"
+        if lang == "en"
+        else "cardiffnlp/twitter-xlm-roberta-base-sentiment"
+    )
+
+    sentiment_pipeline = pipeline("sentiment-analysis", model=model_path)
+    df["sentiment"] = [
+        sentiment["label"] for sentiment in sentiment_pipeline(list(df["message"]))
```
Comment (joweich) [inline chatminer/nlp.py:14]: can we make this more elegant and potentially improve performance by using pandas' [apply](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.apply.html)?

Comment (bdfsaraiva) [reply chatminer/nlp.py]: Of course, see if it's the way you want it, now.
Thanks for the tip.

File: chatminer/nlp.py [hunk 2]
```diff
@@ -3,14 +3,44 @@
...
+        message (str): The input message
+
+        Returns:
+        str: The sentiment of the message
+
+        """
+        try:
+            return sentiment_pipeline(message)[0]["label"]
+        except:
+            print(f"Error processing message: {message}")
```
[the remark this answers is not in the document]
Comment (bdfsaraiva) [reply chatminer/nlp.py:41]: I haven't tested with unfamiliar languages (for me), so it's just for safety.

**Comment by joweich** [comment]
@bdfsaraiva is this PR in draft state for you or do you feel it should be merged already?

**Comment by bdfsaraiva** [comment]
Hi, @joweich,

The package is yours, so the final decision will always be yours.
I think that, as with any feature, there is room for improvement, but that we already have a minimal usable version.

What is missing (as you had suggested) is to add an example in the readme file.

**Comment by joweich** [comment]
What are ideas that you have in mind for improvement? We could add these ideas to a new feature issue and merge this PR for now 👍🏼
I plan to outsource the documentation into a dedicated website, so we don't need to put too much effort into the current readme.

**Comment by bdfsaraiva** [comment]
What are you thinking for the documentation website? [read the docs](https://readthedocs.org/)? [Mkdocs][https://www.mkdocs.org/)? Other?

I can help you with that 🤙🏼

**Comment by joweich** [comment]
> What are you thinking for the documentation website? [read the docs](https://readthedocs.org/)? [Mkdocs](https://www.mkdocs.org/)? Other?
> 
> I can help you with that 🤙🏼

@bdfsaraiva welcome to the main branch! Happy to have you as a contributor to this project! :hugs: 

I plan to go with readthedocs and sphinx as this is what I am used to. Will reach out once I get stuck!

**Pull request merged**